sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -1151,6 +1152,56 @@ class MetastoreDataSourcesSuite extends QueryTest with SQLTestUtils with TestHiv
     }
   }
 
+  test("save API - format hive") {
```

Comment:
Oh sorry I missed this one, what I was asking is, we should only check the provider in `saveAsTable`, so that the `save` API is totally untounched.